selfdrive/camerad/cameras/camera_qcom2.cc

```diff
@@ -1203,10 +1158,8 @@ void cameras_run(MultiCameraState *s) {
 
   LOG(" ************** STOPPING **************");
 
-  err = pthread_join(ae_thread_handle, NULL);
-  assert(err == 0);
+  for (auto &t : threads)
+    t.join();
 
   cameras_close(s);
```

Comment:
wait all threads exit before call cameras_close

Reply:
submitted another PR to fix this problem https://github.com/commaai/openpilot/pull/19555